projects/plugins/jetpack/functions.compat.php

```diff
@@ -27,71 +33,95 @@ function jetpack_get_youtube_id( $url ) {
 	$url = wp_parse_url( $url );
 	$id  = false;
 
-	if ( ! isset( $url['query'] ) )
+	if ( ! isset( $url['query'] ) ) {
 		return false;
+	}
 
 	parse_str( $url['query'], $qargs );
 
-	if ( ! isset( $qargs['v'] ) && ! isset( $qargs['list'] ) )
+	if ( ! isset( $qargs['v'] ) && ! isset( $qargs['list'] ) ) {
 		return false;
+	}
 
-	if ( isset( $qargs['list'] ) )
+	if ( isset( $qargs['list'] ) ) {
 		$id = preg_replace( '|[^_a-z0-9-]|i', '', $qargs['list'] );
+	}
 
-	if ( empty( $id ) )
+	if ( empty( $id ) ) {
 		$id = preg_replace( '|[^_a-z0-9-]|i', '', $qargs['v'] );
+	}
 
 	return $id;
 }
 
-if ( !function_exists( 'youtube_sanitize_url' ) ) :
-/**
-* Normalizes a YouTube URL to include a v= parameter and a query string free of encoded ampersands.
-*
-* @param string $url
-* @return string The normalized URL
-*/
-function youtube_sanitize_url( $url ) {
-	$url = trim( $url, ' "' );
-	$url = trim( $url );
-	$url = str_replace( array( 'youtu.be/', '/v/', '#!v=', '&amp;', '&#038;', 'playlist' ), array( 'youtu.be/?v=', '/?v=', '?v=', '&', '&', 'videoseries' ), $url );
-
-	// Replace any extra question marks with ampersands - the result of a URL like "https://www.youtube.com/v/9FhMMmqzbD8?fs=1&hl=en_US" being passed in.
-	$query_string_start = strpos( $url, "?" );
-
-	if ( false !== $query_string_start ) {
-		$url = substr( $url, 0, $query_string_start + 1 ) . str_replace( "?", "&", substr( $url, $query_string_start + 1 ) );
+if ( ! function_exists( 'youtube_sanitize_url' ) ) :
+	/**
+	 * Normalizes a YouTube URL to include a v= parameter and a query string free of encoded ampersands.
+	 *
+	 * @param string $url YouTube URL.
+	 * @return string The normalized URL
+	 */
+	function youtube_sanitize_url( $url ) {
+		$url = trim( $url, ' "' );
+		$url = trim( $url );
+		$url = str_replace( array( 'youtu.be/', '/v/', '#!v=', '&amp;', '&#038;', 'playlist' ), array( 'youtu.be/?v=', '/?v=', '?v=', '&', '&', 'videoseries' ), $url );
+
+		// Replace any extra question marks with ampersands - the result of a URL like "https://www.youtube.com/v/9FhMMmqzbD8?fs=1&hl=en_US" being passed in.
+		$query_string_start = strpos( $url, '?' );
+
+		if ( false !== $query_string_start ) {
+			$url = substr( $url, 0, $query_string_start + 1 ) . str_replace( '?', '&', substr( $url, $query_string_start + 1 ) );
+		}
+
+		return $url;
 	}
-
-	return $url;
-}
 endif;
 
 /**
- * Merge in three string helper functions from WPCOM.
+ * Merge in three string helper functions from WPCOM to make working with strings easier.
  *
  * @see WPCOM/wp-content/mu-plugins/string-helpers.php
  */
 if ( ! function_exists( 'wp_startswith' ) ) :
+	/**
+	 * Filters strings that start with a specific substring.
+	 *
+	 * @param var    $haystack String we are filtering.
+	 * @param string $needle The substring we are looking for.
+	 */
```

Comment:
May as well document the return values when we're adding doc comments.
```suggestion
	 * @return bool
	 */
```